lib/data/model/NewStageEntity.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:farmsmart_flutter/data/model/article_entity.dart';
+
+class NewStageEntity {
+  final String id;
+  final ArticleEntity article;
+  final DateTime started;
+  final DateTime ended;
+
+  NewStageEntity({String id, ArticleEntity article, DateTime started, DateTime ended}) : this.id = id, this.article = article, this.started = started, this.ended = ended;
```

Comment:
Missing comma before `}` so in this way the IDE can format that. There're some places in the PR where this happens, I haven't added comments at each one, if you want I can go through them and add the missing comma at the end.

```suggestion
  NewStageEntity({
    String id,
    ArticleEntity article,
    DateTime started,
    DateTime ended,
  })  : this.id = id,
        this.article = article,
        this.started = started,
        this.ended = ended;
```

Reply:
👍